src/app.js

```diff
@@ -515,12 +516,36 @@ var run = function () {
       })
   }
 
+  function handleSwarmImport (url, cb) {
+    swarmgw.get(url, function (err, content) {
+      // retry if this failed and we're connected via RPC
+      if (err && !executionContext.isVM()) {
```

Comment:
Unfortunately we have to remove the RPC support because it will be removed, I have been told by the Swarm team.